doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -73,6 +73,7 @@ nRF9160: Asset Tracker v2
 * Added new documentation for Asset Tracker v2 :ref:`asset_tracker_v2_modem_module`.
 * Added support for A-GPS filtered ephemerides.
 * Added new documentation for :ref:`asset_tracker_v2_util_module` and :ref:`api_modules_common`.
+* Added test for :ref:`asset_tracker_v2_ui_module`.
```

Comment:
Just a question: Is this necessary? If so, please change the text to "Added unit test..."

Reply:
Well, its not strictly necessary. But it doesn't hurt either 😄